packages/keycodes/src/index.js

```diff
@@ -53,7 +53,7 @@ const modifiers = {
 	alt: () => [ ALT ],
 	ctrlShift: () => [ CTRL, SHIFT ],
 	shift: () => [ SHIFT ],
-	shiftAlt: () => [ SHIFT, ALT ],
+	shiftAlt: ( _isApple ) => _isApple() ? [ 'Shift', 'Alt' ] : [ 'Shift', 'Control' ],
```

Comment:
It seems like we should continue using constant as in other places.

Reply:
For sure, have update this.